docs/docusaurus/docs/core/define_expectations/organize_expectation_suites.md

```diff
@@ -0,0 +1,128 @@
+
+5. Add the Expectation to the Expectation Suite.
+
+   An Expectation Suite's `add_expectation(...)` method takes in an instance of an Expectation and adds it to the Expectation Suite's configuraton: 
+
+   ```python title="Python input"
+   suite.add_expectation(expectation)
+   ```
+   
+   You can also instantiate an Expectation at the same time as you add it to an Expectation Suite:
+
+   ```python title="Python input"
+   import great_expectations.expectations as gxe
+   
+   suite.add_expectation(gxe.ExpectColumnValuesToNotBeNull(column="pickup_datetime"))
+   ```
+
+   If you have a configured Data Source, Data Asset, and Batch Definition you can test your Expectation before adding it to your Expectation Suite.  To do this see [Test an Expectation](./test_an_expectation.md).
+
+   However, if you test and modify an Expectation _after_ you have added it to your Expectation Suite you must explicitly save those modifications before they will be pushed to the Expectation Suite itself:
+
+   ```python title="Python input"
+   expectation.column = "pickup_location_id"
+   expectation.save()
+   ```
+
+6. Continue to create and add additional Expectations
+   
+   Repeat the process of creating, testing, and adding Expectations to your Expectation Suite until the Expectation Suite adequately describes your data's ideal state.
+
+7. Add the Expectation Suite to your Data Context
+
+   Once you have finalized the contents of your Expectation Suite you should save it to your Data Context:  
+
+   ```python title="Python input"
+   suite = context.suites.add(suite)
+   ```
+
+   With a File or GX Cloud Data Context your saved Expectation Suite will be available between Python sessions.  You can retrieve your Expectation Suite from your Data Context with the following code:
+
+   ```python title="Python input"
+   existing_suite_name = "my_expectation_suite"  # replace this with the name of your Expectation Suite
+   suite = context.suites.get(name=existing_suite_name)
+   ```
+
+</TabItem>
+
+<TabItem value="sample_code" label="Sample code">
+
+```python title="Python input"
+import great_expectations as gx
+from great_expectations.core.expectation_suite import ExpectationSuite
+import great_expectations.expectations as gxe
+
+context = gx.get_context()
+expectation = gxe.ExpectColumnValuesToNotBeNull(column="passenger_count")
+
+# Create an Expectation Suite
+suite_name = "my_expectation_suite"
+suite = ExpectationSuite(name=suite_name)
+
+# Add a previously created Expectation to the Expectation Suite
+suite.add_expectation(expectation)
+
+# Add an Expectation to the Expectation Suite when it is created
+suite.add_expectation(gxe.ExpectColumnValuesToNotBeNull(column="pickup_datetime"))
+
+# Update the configuration of an Expectation, then push the changes to the Expectation Suite
+expectation.column = "pickup_location_id"
+expectation.save()
```

Comment:
Does the Expectation Suite need to be added to the context's list of suites? If not, it would be worth mentioning this explicitly.

Reply:
That is a good question.  I'll look into that and clarify in the doc.